CHANGES.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.2.0] - 2021-11-09
+
+### Added
+
+- Depositing ETH from Ethereum to Aurora now allows an `0x` prefix on the recipeint address by [@joshuajbouw]. ([#337](https://github.com/aurora-is-near/aurora-engine/pull/337))
```

Comment:
Spelling of `recipient`

Reply:
Fixed in https://github.com/aurora-is-near/aurora-engine/pull/340/commits/8b205bae57760e72b13f63a63b338f10aebbfb27